src/app/utilities/SchemaUtilities.ts

```diff
@@ -11,27 +14,28 @@ import { Dict } from "app/Types";
  * @returns Dictionary containing the generated classes and terms list
  */
 const ExtractClassesAndTermsFromSchema = async (schema: Dict) => {
-    const classesList: { label: string, value: string }[] = [];
-    const termsList: { label: string, options: { label: string, value: string }[] }[] = [];
+    const classesList: { key: string, label: string, value: string }[] = [];
+    const termsList: { label: string, options: { key: string, label: string, value: string }[] }[] = [];
 
     /**
-     * 
+     * Function to push to the classes list
      * @param classOption 
      */
-    const PushToClassesList = (classOption: { label: string, value: string }) => {
+    const PushToClassesList = (classOption: { key: string, label: string, value: string }) => {
```

Comment:
Well it could, but I am pushing it to a recursive function, so it will reuse this function every time it loops and since it refers to the level of the function it is defined in, I can update the array `classesList` in there